entity-types/ext-connector/definition.yml

```diff
@@ -0,0 +1,26 @@
+domain: EXT
+type: CONNECTOR
+synthesis:
+  rules:
+  - identifier: service
+    name: service
+    conditions:
+    - attribute: account
+      present: true
+    - attribute: region
+      present: true
+    - attribute: application
+      value: "sherpa"
```

Comment:
I guess this is for testing?
The conditions seem too generic right now. Is there any other we can add? Like specific eventTypes or similar?

Reply:
IF you think I need another attribute..etc in order to ensure uniqueness I will.   I am using this to test a few things out , and if it works, with my very generic test data (I am pushing with script) which sort of mimics customers data today.  If everything works, I will show this to customer, and go from there.

Reply:
that is why I used application = sherpa,,